### PR TITLE
docs: add sample for harvest vm

### PR DESCRIPTION
**Pull request by helayoty**

### CODE REVIEW

File: examples/harvest-vm/README.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+
```
Comment (aramase) [inline examples/harvest-vm/README.md:1]: nit: remove new line

File: examples/harvest-vm/README.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+# Running Placement Policy Scheduler Plugins in Harvest VM cluster
+
```
Comment (aramase) [inline examples/harvest-vm/README.md:3]: nit: remove extra new line

File: examples/harvest-vm/README.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+## AKS demo
+
+#### 1. Create [AKS](https://docs.microsoft.com/en-us/azure/aks/) cluster
```
Comment (aramase) [inline examples/harvest-vm/README.md:7]:
```suggestion
#### 1. Create an [AKS](https://docs.microsoft.com/en-us/azure/aks/) cluster with harvest vm
```

File: examples/harvest-vm/README.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    --resource-group harvestvm \
+    --name harvestaks \
+    --vm-set-type VirtualMachineScaleSets \
+    --node-vm-size Harvest_E2s_v3
+    --node-count 3 \
+    --generate-ssh-keys \
+    --load-balancer-sku standard
+```
+
+When the cluster is ready, use the az aks get-credentials command to get the cluster credentials for use with kubectl:
```
Comment (aramase) [inline examples/harvest-vm/README.md:24]:
```suggestion
Run `az aks get-credentials` command to get access credentials for the cluster:
```

File: examples/harvest-vm/README.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+```
+az aks get-credentials --resource-group harvestvm --name harvestaks
+```
+
+Add a second node pool that runs 3 nodes:
```
Comment (aramase) [inline examples/harvest-vm/README.md:30]:
```suggestion
Add a second node pool with 3 nodes:
```

File: examples/harvest-vm/README.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+  name: harvest-stric-must
+spec:
+  weight: 100
+  enforcementMode: Strict
+  podSelector:
+    matchLabels:
+      app: nginx
+  nodeSelector:
+    matchLabels:
+      # instansce type can be one of the following( Harvest_E2s_v3, Harvest_E4s_v3, Harvest_E8s_v3)
```
Comment (aramase) [inline examples/harvest-vm/README.md:78]:
```suggestion
      # instance type can be one of the following( Harvest_E2s_v3, Harvest_E4s_v3, Harvest_E8s_v3)
```

Comment (aramase) [reply examples/harvest-vm/README.md:78]: Can we also add a step for how to get this node label for the existing nodes?

File: examples/harvest-vm/README.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+nginx-xc2cr   1/1     Running   0          56s   10.244.1.10   aks-agentpool-33997223-vmss000002   <none>           <none>
+nginx-xvqbb   1/1     Running   0          56s   10.244.7.5    aks-normalvms-23099053-vmss000000   <none>           <none>
+nginx-dmb4h   1/1     Running   0          56s   10.244.7.6    aks-normalvms-23099053-vmss000000   <none>           <none>
+nginx-skzrk   1/1     Running   0          56s   10.244.8.6    aks-normalvms-23099053-vmss000001   <none>           <none>
+nginx-hrznh   1/1     Running   0          56s   10.244.8.5    aks-normalvms-23099053-vmss000001   <none>           <none>
+nginx-6c87l   1/1     Running   0          56s   10.244.6.6    aks-normalvms-23099053-vmss000002   <none>           <none>
+nginx-f9mm2   1/1     Running   0          56s   10.244.6.5    aks-normalvms-23099053-vmss000002   <none>           <none>
+```
+
+We will find the nodes which carry the same node selector defined in the strict-must `PlacementPolicy` have been assigned 40% of the workload as defined with `targetSize`.
```
Comment (aramase) [inline examples/harvest-vm/README.md:127]:
```suggestion
We will find the nodes which carry the same node selector defined in the harvest-strict-must `PlacementPolicy` have been assigned 40% of the workload as defined with `targetSize`.
```

File: examples/harvest-vm/README.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+
+```
+</details><br/>
+
+#### 3. Deploy a `PlacementPolicy` CRD
+
+```yaml
+kind: PlacementPolicy
+metadata:
+  name: harvest-stric-must
```
Comment (aramase) [inline examples/harvest-vm/README.md:69]:
```suggestion
  name: harvest-strict-must
```

File: examples/harvest-vm/README.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+  policy:
+    action: Must
+    targetSize: 40%
+```
+
+<details>
+<summary>Result</summary>
+
+```
+placementpolicy.placement-policy.scheduling.x-k8s.io/strict-must created
```
Comment (aramase) [inline examples/harvest-vm/README.md:89]:
```suggestion
placementpolicy.placement-policy.scheduling.x-k8s.io/harvest-strict-must created
```

File: examples/harvest-vm/README.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+nginx-hrznh   1/1     Running   0          56s   10.244.8.5    aks-normalvms-23099053-vmss000001   <none>           <none>
+nginx-6c87l   1/1     Running   0          56s   10.244.6.6    aks-normalvms-23099053-vmss000002   <none>           <none>
+nginx-f9mm2   1/1     Running   0          56s   10.244.6.5    aks-normalvms-23099053-vmss000002   <none>           <none>
+```
+
+We will find the nodes which carry the same node selector defined in the strict-must `PlacementPolicy` have been assigned 40% of the workload as defined with `targetSize`.
+
+#### 6. Clean up
+
+- Delete [AKS](https://docs.microsoft.com/en-us/azure/aks/)
```
Comment (aramase) [inline examples/harvest-vm/README.md:131]:
```suggestion
- Delete [AKS](https://docs.microsoft.com/en-us/azure/aks/) cluster
```

File: examples/harvest-vm/v1alpha1_placementpolicy_harvestvm.yml [hunk 11]
```diff
@@ -0,0 +1,17 @@
+apiVersion: placement-policy.scheduling.x-k8s.io/v1alpha1
+kind: PlacementPolicy
+metadata:
+  name: harvest-stric-must
```
Comment (aramase) [inline examples/harvest-vm/v1alpha1_placementpolicy_harvestvm.yml:4]:
```suggestion
  name: harvest-strict-must
```

File: examples/harvest-vm/README.md (hunk 6 above)
Comment (helayoty) [reply examples/harvest-vm/README.md]: Please let me know what you think [step 3](https://github.com/Azure/placement-policy-scheduler-plugins/pull/55/files/493684b947b9964145ca36748ef44efb80bdd84a#diff-5112de02c93dd5f001ff64f0e5b79191c18bd697fffed12283fc55e8a1f70271R64)

File: examples/harvest-vm/README.md [hunk 12]
```diff
@@ -0,0 +1,156 @@
+
```
Comment (aramase) [inline examples/harvest-vm/README.md:1]:
```suggestion
```

File: examples/harvest-vm/README.md [hunk 13]
```diff
@@ -0,0 +1,156 @@
...
+clusterrolebinding.rbac.authorization.k8s.io/pp-plugins-scheduler-as-kube-scheduler created
+serviceaccount/pp-plugins-scheduler created
+deployment.apps/pp-plugins-scheduler created
+
+```
+</details><br/>
+
+#### 3. Choose node label
+
+To identify the harvest vms, we can use node label `node.kubernetes.io/instance-type` and set the value by choosing the harvest vm size by runing the following command.
```
Comment (aramase) [inline examples/harvest-vm/README.md:65]:
```suggestion
To identify the harvest vms, we can use the instance type node label: `node.kubernetes.io/instance-type: <vm size>`.  Run the following command to get the vm size:
```

**Review by aramase — APPROVED** [review]

**Pull request merged**